### ISSUE TITLE
Where is the version for windows 32 bits?

### ISSUE THREAD

**Issue by lordk1977** [kwsch/PKHeX#3942]
Good morning! Good afternoon! Goodnight! there will no longer be a 32-bit version for Windows. Many people have the system in 32, myself. I tried to use it to save pokemon violet, but without success. I tried to look for an older version of the program that is for 32 bits, but I didn't find it. Thank you very much in advance

**Comment by ReignOfComputer** [comment]
Pretty sure 32-bit Windows platforms makes up <1% of the market share, coming in at 0.08% of all platform users as per Steam's June 2023 statistics. You might want to upgrade your PC rather than have PKHeX support it, or self-compile the code.

**Comment by lordk1977** [comment]
Thank you very much for answering. Do you have the latest 32-bit version that works for scarlet and violet saves that I can download and use? Thank you very much.

**Comment by kwsch** [comment]
No; compile the application yourself by changing the csproj settings to build as 32bit instead, or use a different (64bit) computer.
